### DISCUSSION TITLE
iOS yaml workflow failing after switching from editor

### DISCUSSION THREAD

**Discussion by thecoder-co** [codemagic-ci-cd/codemagic-docs#2620] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

```ios-workflow:
    name: iOS Workflow
    instance_type: mac_mini_m1
    max_build_duration: 120
    integrations:
      app_store_connect: CodeMagicAPIKey
    environment:
      ios_signing:
        distribution_type: app_store
        bundle_identifier: ...
      vars:
        APP_ID: ... # <-- Put your APP ID here
      flutter: stable
      xcode: latest # <-- set to specific version e.g. 14.3, 15.0 to avoid unexpected updates.
      cocoapods: default
    scripts:
      - name: Set up code signing settings on Xcode project
        script: |
          xcode-project use-profiles
      - name: Get Flutter packages
        script: |
          flutter packages pub get
      - name: Install pods
        script: |
          find . -name "Podfile" -execdir pod install \;
        ignore_failure: true
      - name: Flutter build ipa and automatic versioning
        script: |
          flutter build ipa --release
    artifacts:
      - build/ios/ipa/*.ipa
      - /tmp/xcodebuild_logs/*.log
      - flutter_drive.log
    publishing:
      # See the following link for details about email publishing - https://docs.codemagic.io/publishing-yaml/distribution/#email
      email:
        recipients:
          - ...
        notify:
          success: true # To receive a notification when a build succeeds
          failure: true # To not receive a notification when a build fails
      app_store_connect:
        auth: integration

        submit_to_testflight: true
        submit_to_app_store: false
```
       Used this as the yaml iOS workflow 

### Expected results

Build and push to testflight

### Actual results

Fails immediately with error `No matching profiles found for bundle identifier "<bundle-id>" and distribution type "app_store"`

### Build id (optional)

6682ad1a144e45ec3b40e017

**Comment by budescode** [comment]
Hi, I'm facing the same issue, were you able to get a fix?
